users/models.py

```diff
@@ -191,17 +197,33 @@ class Profile(TimestampedModel):
 
     user = models.OneToOneField(User, on_delete=models.CASCADE, related_name="profile")
 
-    gender = models.CharField(max_length=10, choices=GENDER_CHOICES)
-    birth_year = models.IntegerField()
+    gender = models.CharField(
+        max_length=10, blank=True, choices=GENDER_CHOICES, default=""
+    )
+    birth_year = models.IntegerField(null=True, blank=True)
 
-    company = models.CharField(max_length=128)
-    job_title = models.CharField(max_length=128)
-    industry = models.CharField(max_length=60, blank=True)
-    job_function = models.CharField(max_length=60, blank=True)
+    company = models.CharField(max_length=128, blank=True, default="")
+    job_title = models.CharField(max_length=128, blank=True, default="")
+    industry = models.CharField(max_length=60, blank=True, default="")
+    job_function = models.CharField(max_length=60, blank=True, default="")
     company_size = models.IntegerField(
         null=True, blank=True, choices=COMPANY_SIZE_CHOICES
     )
     years_experience = models.IntegerField(
         null=True, blank=True, choices=YRS_EXPERIENCE_CHOICES
     )
-    leadership_level = models.CharField(max_length=60, blank=True)
+    leadership_level = models.CharField(max_length=60, blank=True, default="")
+
+    @property
+    def is_complete(self):
+        """Returns True if the profile is complete"""
+        return (
+            bool(self.gender)
+            and bool(self.birth_year)
+            and bool(self.company)
+            and bool(self.job_title)
+        )
```

Comment:
Take or leave: can be rewritten as `return all((self.gender, self.birth_year, self.company, self.job_title))`